homeassistant/components/input_select/__init__.py

```diff
@@ -65,20 +88,78 @@ def _cv_input_select(cfg):
 RELOAD_SERVICE_SCHEMA = vol.Schema({})
 
 
-async def async_setup(hass, config):
+class InputSelectStorageCollection(collection.StorageCollection):
```

Comment:
Please move this below `async_setup`. It's our standard to start with `async_setup` at the top.

Reply:
done in https://github.com/home-assistant/home-assistant/pull/30604/commits/834e0aa603127feec46db6d99a1e733648f992c4